prow/config.yaml

```diff
@@ -16193,7 +16193,7 @@ periodics:
   agent: kubernetes
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bigquery:0.2
+    - image: gcr.io/k8s-testimages/bigquery:0.3
```

Comment:
Update tag to new format?

Reply:
I'll update this with a new commit once I've squashed review commits.